src/mono/wasm/templates/templates/console/app-support.mjs

```diff
@@ -47,11 +47,26 @@ function set_exit_code(exit_code, reason) {
     }
 
     if (App && App.INTERNAL) {
-        App.INTERNAL.mono_wasm_exit(exit_code);
+        let _flush = function(_stream) {
+             return new Promise((resolve, reject) => {
+                 setTimeout(reject, 30000);
+                 _stream.write('', function() { resolve () });
+                 _stream.on('error', (error) => reject(error));
```

Comment:
shouldn't you do this before the write instead of after, since .write could cause an error?

Reply:
Should we remove the timeout completely for console template, IOW, user apps?

the other usage of the timeout is in tests, which should be useful.

Reply:
I think removing it for apps is probably reasonable